src/tasks/tasks.py

```diff
@@ -45,13 +45,14 @@ def sentence_to_text_field(sent: Sequence[str], indexers: Any):
     return TextField(list(map(Token, sent)), token_indexers=indexers)
 
 
-def atomic_tokenize(sent: str, atomic_tok: str, nonatomic_toks: List[str], max_seq_len: int):
+def atomic_tokenize(sent: str, atomic_tok: str, nonatomic_toks: List[str], max_seq_len: int,
+                    tokenizer_name: str):
```

Comment:
nit: be consistent between `tokenizer` and `tokenizer_name`.

I guess this is partly my fault for calling it `args.tokenizer`, but can we use `tokenizer_name` for things that are strings, and `tokenizer` for the actual object or function?

Reply:
:+1: I think I addressed this everywhere